packages/server/src/server.ts

```diff
@@ -161,17 +166,28 @@ const logAndHandleResponse = async (methodName, methodFunction) => {
   }
 };
 
+/**
+ * Generates random trace id for requests.
+ * 
+ * returns: integer
+ */
+ const generateRequestId = () => {
+  const id = crypto.randomInt(REQUEST_ID_MAX_VALUE);
+  return id;
+};
+
+
 /**
  * returns: false
  */
-rpc.use('net_listening', async () => {
+rpc.use('net_listening',async () => {
```

Comment:
nit: revert this change here and below
```suggestion
rpc.use('net_listening', async () => {
```